packages/rum-core/src/rumEvent.types.ts

```diff
@@ -381,6 +381,16 @@ export type RumErrorEvent = CommonProperties &
         readonly path?: string
         [k: string]: unknown
       }
+      /**
+       * Content Security Violation properties
+       */
+      readonly csp?: {
+        /**
+         * In the context of CSP errors, indicates how the violated policy is configured to be treated by the user agent.
+         */
+        readonly disposition?: 'enforce' | 'report'
+        [k: string]: unknown
+      }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏given that there is no update of the rum-events-format reference in the PR, I am guessing that you have updated this file manually.
You should run `yarn rum-events-format:sync` to update all the event format types with the latest changes.